### PR TITLE
Changes the practice beam name to "practice laser"

### LINKED ISSUE #6729: Practice Laser shows up identically to the real laser in combat logs
[10:01:43] Has been shot by Blessed H. Legionare(blessedheretic) with The laser = null 
[10:02:09] Has been shot by Blessed H. Legionare(blessedheretic) with The laser = null

One is with the practice laser and one is with a real one. The projectile should probably be renamed.

### PR DESCRIPTION
**Pull request by tkdrg**
Fixes #6729

### CODE REVIEW

**Comment by Lo6a4evskiy** [comment]
>But that removes all the fun of spooking people with it, if anything you could override the logging here so the message is 'laser' but the logged message is 'practice laser'

Seriously, you only see the name of the gun after you get scared by huge red laser hit

**Comment by paprka** [comment]
It's fine.

**Comment by phil235** [comment]
I think this conflicts with Carn's PR. Can't merge yet.

File: code/modules/projectiles/projectile/beams.dm [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	eyeblur = 2
 
 /obj/item/projectile/practice
```
Comment (Anonus) [inline code/modules/projectiles/projectile/beams.dm]: why is this not a subtype of /beam

Comment (tkdrg) [inline code/modules/projectiles/projectile/beams.dm]: That would make it inherit beam damage.

Comment (paprka) [inline code/modules/projectiles/projectile/beams.dm]: Use nodamage = 1 i think

**Comment by tkdrg** [comment]
Alright, I repathed it. Should be merge-ready.

**Pull request merged**